sklearn/datasets/base.py

```diff
@@ -290,6 +290,65 @@ def load_iris():
                                 'petal length (cm)', 'petal width (cm)'])
 
 
+def load_wdbc():
+    """Load and return the breast cancer wisconcin dataset (classification)
+    
+    =================   ==============
+    Classes                          2
+    Samples per class    212(M),357(B)
+    Samples total                  569
+    Dimensionality                  30
+    Features            real, positive
+    =================   ==============    
+    Returns
+    -------
+    data : Bunch
+        Dictionary-like object, the interesting attributes are:
+        'data', the data to learn, 'target', the classification labels,
+        'target_names', the meaning of the labels, 'feature_names', the
+        meaning of the features, and 'DESCR', the
+        full description of the dataset.
+
+    Examples
+    --------
+    Let's say you are interested in the samples 10, 50, and 85, and want to
+    know their class name.
+
+    >>> from sklearn.datasets import load_wdbc
+    >>> data = load_wdbc()
+    >>> data.target[[10, 50, 85]]
+    array([0, 1, 0])
+    >>> list(data.target_names)
+    ['malignant', 'benign']
+    """
+    module_path = dirname(__file__)
+    with open(join(module_path, 'data', 'wdbc.csv')) as csv_file:
+        data_file = csv.reader(csv_file)
+        temp = next(data_file)
+        n_samples = int(temp[0])
+        n_features = int(temp[1])
+        target_names = np.array(temp[2:4])
+        data = np.empty((n_samples, n_features))
+        target = np.empty((n_samples,), dtype = np.int)
+        feature_names = ['mean radius', 'mean texture', 'mean perimeter', 'mean area', 'mean smoothness', 
```

Comment:
[PEP8](https://www.python.org/dev/peps/pep-0008/) is a list of guidelines for Python coding.
You can install the pep8 (or flake8) package in order to check automatically the compliance to PEP8 guidelines.
To do so, you can type in your terminal `pip install flake8` or `conda install flake8` if you are using Anaconda.
Then the command `flake8 path/to/your/code.py` will list the corrections to make.